ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlanningTools.java

```diff
@@ -18,6 +21,22 @@
 
 public class ContinuousPlanningTools
 {
+   public static double getDistanceFromRobotToGoalPoseOnXYPlane(Vector3DBasics robotPositionInWorld, SideDependentList<FramePose3D> goalPoses)
```

Comment:
Need to know how far away we are from a goal pose "waypoint" to see if we should start going to wards the next one or not. Maybe there is a better method that already does this

Reply:
Pass in robotPositionInWorld as a Point2DReadOnly or a Point3DReadOnly. Then you don't need lines 34 and 35. You can then use the `distanceXY` method, instead of `distance.`

Reply:
Ya thats nice, passed in a Point3DReadOnly and removed those lines